### ISSUE TITLE
Missing `LICENSE` file

### ISSUE THREAD

**Issue by Tyler887** [cyweb/hammer#53]
There's no license for this project. The default is *copyright*.

I'd suggest you'd release this project under the GPLv3 (with the "any later version" clause) so if anyone makes changes to your project, they must also distribute their changes under the GPLv3 or any later version.

**Comment by Nexuzzzz** [comment]
yeah, else you're gonna have people steal your code and use it in closed source apps
